test/parallel/test-pipe-outgoing-message-data-emitted-after-ended.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+const OutgoingMessage = require('_http_outgoing').OutgoingMessage;
+
+const server = http.createServer(common.mustCall(function(req, res) {
+  console.log('Got a request, piping an OutgoingMessage to it.');
+  const outgointMessage = new OutgoingMessage();
+  outgointMessage.pipe(res);
```

Comment:
`OutgoingMessage` should really be write-only, and not be piped. The fact that `OutgoingMessage` has a `pipe`  method is related to Node.js history, and it _should not be used_. Why Does this use case matter to you?